languagetool-server/pom.xml

```diff
@@ -124,7 +124,9 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <!--
```

Comment:
I think we just can remove this instead of commenting out. Will stay in the git history.